components/avatar/avatar.css

```diff
@@ -0,0 +1,27 @@
+.avatar-small {
+  vertical-align: middle;
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+}
+
```

Comment:
Using Pixles in height and width. it would give overflow problems in responsive website.